components/brave_new_tab_ui/components/default/braveNews/index.tsx

```diff
@@ -119,15 +121,16 @@ export default function BraveNewsSection (props: Props) {
         ref={loadDataTrigger}
         style={{ position: 'sticky', top: '100px' }}
       />
-      { !optedIn &&
-      <>
-        <CardOptIn onOptIn={() => setOptedIn(true)} onDisable={() => setShowToday(false)} />
-      </>
+      {!optedIn &&
+        <>
+          <CardOptIn onOptIn={() => setOptedIn(true)} onDisable={() => setShowToday(false)} />
+        </>
       }
-      { shouldDisplayContent &&
-      <React.Suspense fallback={(<CardLoading />)}>
-        <Content {...props} />
-      </React.Suspense>
+      {shouldDisplayContent && (defaultState.featureFlagBraveNewsFeedV2Enabled
+        ? <FeedV2 />
```

Comment:
Lazy-load feedv2 and then you don't really need to lazy-load anything in FeedV2.tsx